test/utility/generateToken.js

```diff
@@ -0,0 +1,31 @@
+import should from 'should';
+import jwt from 'jsonwebtoken';
+import generateToken from '../../config/generateToken';
+
+/* global dotenv */
+
+dotenv.config();
+
+const jwtSecret = process.env.JWT_TOKEN || 'SECRET';
+
+let user;
+
+describe('GenerateToken', () => {
+  before(() => {
```

Comment:
remove before block and just assign const user to 
`{
     username: 'testUser'
 }`